passes/spec_source.nim

```diff
@@ -20,11 +20,12 @@ type
     Params
     ProcDecl
     TypeDecl
+    Exprs
```

Comment:
Minor nit-pick: I'd move `Exprs` to after `FieldAccess`, so that it's grouped together with the other non-void expressions.